src/main/java/com/google/javascript/gents/TypeConversionPass.java

```diff
@@ -229,6 +261,42 @@ public void visit(NodeTraversal t, Node n, Node parent) {
     }
   }
 
+  /** Renames the inner typedefs to the new top level typedefs */
+  private class typeAliasConverter extends AbstractPostOrderCallback {
```

Comment:
Fixed the class name.

`TypeAliasConverter` is non-static because `typesToRename` from the enclosing class is used here.